### DISCUSSION TITLE
Document of specification of MSE format

### DISCUSSION THREAD

**Discussion by fpluquet** [moosetechnology/Moose#2497] [Q&A] [answered]
I'm looking for a document of specification of MSE format to load MSE file in nodejs. Is a such document exist ? Thanks !

**Comment by Ducasse** [comment]
Hi frederic

this pdf is the only doc we have.
[Duca11c-Cutter-deliverable22-MSE-FAMIX30.pdf](https://github.com/moosetechnology/Moose/files/11714296/Duca11c-Cutter-deliverable22-MSE-FAMIX30.pdf)

**Comment by badetitou** [comment, marked as answer]
There is now a documentation page in our website

https://modularmoose.org/users/file-format/
